hist/hist/inc/TH1.h

```diff
@@ -311,6 +311,9 @@ class TH1 : public TNamed, public TAttLine, public TAttFill, public TAttMarker {
    virtual TArrayD *GetSumw2() {return &fSumw2;}
    virtual const TArrayD *GetSumw2() const {return &fSumw2;}
    virtual Int_t    GetSumw2N() const {return fSumw2.fN;}
+           /// This function returns the Standard Deviation (Sigma) of the distribution (not RMS).
```

Comment:
(not RMS) -> not the Root Mean Square (RMS)., 
The name "RMS" is been often used as a synonym for the Standard Deviation and it was introduced many years ago (Hbook/PAW times).